app/models.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
+from sqlmodel import (
+  SQLModel, 
+  Field, 
+  create_engine
+)
+
+engine = create_engine('sqlite:///database.db')
+
+class Tutor(SQLModel, table=True):
+  id: Optional[int] = Field(default=None, primary_key=True)
+  title: str
+  ranking: int
+  name: str
+  photo: str
+  isVerified: bool 
```

Comment:
The database optimization we can do is to put string fields at the end of the table
In the future we can make a structure where we separate the engine from the tables
```suggestion
  id: Optional[int] = Field(default=None, primary_key=True)
  ranking: int
  isVerified: bool 
  title: str
  name: str
  photo: str
```